libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -326,6 +326,76 @@ TEST(CppInterface, PreprocessingOnlyModeSecondOverridesDisassemblyMode) {
               HasSubstr("void main(){ }"));
 }
 
+TEST(CppInterface, WarningsOnLine) {
+  shaderc::Compiler compiler;
+  shaderc::CompileOptions options;
+  // By default the compiler will emit a warning on line 2 complaining
+  // 'float' as a deprecated attribute in version 130.
+  const std::string kDeprecatedAttributeShader =
+      "#version 130\n"
+      "attribute float x;\n"
+      "void main() {}\n";
+  shaderc::SpvModule result_suppress_warnings_on_line =
```

Comment:
Also, why "suppress" in the name?  I don't understand what's being suppressed here.